AdminCore.Common/Authorization/AdminCoreRolesHandler.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AdminCore.Constants;
+using AdminCore.Constants.Enums;
+using Microsoft.AspNetCore.Authorization;
+
+namespace AdminCore.Common.Authorization
+{
+    public class AdminCoreRolesHandler : AuthorizationHandler<AdminCoreRolesRequirement>
+    {
+        private readonly Predicate<Claim> claimPredicate = claim => claim.Type == UserDetailsConstants.Role;
+
+        protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, AdminCoreRolesRequirement requirement)
```

Comment:
is this behaviour tested?

Reply:
The AuthorizationHandler is part of the dotnet core authorization middleware so I can't really test it and this particular method is a protected one.